modules/ROOT/pages/parse-template-reference.adoc

```diff
@@ -70,15 +70,16 @@ include::{examplesdir}/responseHtml.template[]
 include::{examplesdir}/parse-template-reference.xml[]
 ----
 
-To use the application in this example, you must send an HTTP request with a URL that includes an ID query parameter, such as +`http://localhost:8081/getEmployee?id=12345`+.
+To use the application in this example, you must send an HTTP request with a URL that includes an ID query parameter, such as `http://localhost:8081/getEmployee?id=12345`.
 
 ==== Special Characters
 
-Parse Template supports the use of nested expression and literals inside expressions. There are some characters for special uses: 
-* An expression, which begins with the sequence `#[`, can contain nested expressions. Inside an expression, the character `[` can define the start of an expression. The  `[` does not need a preceding `#`. `[` is treated as a special character inside an expression.
-* Inside an expression, single quotes are used both to quote text and to treat text as a literal. So, the expression `#['[']` is a valid expression that evaluates to `[`. Outside expressions, single quotes are used as literals. 
-* To avoid using a single quote to quote text inside an expression, you must add a backslash to escape it. For example: `#['this isn\'t an expression']`.
-* Outside an expression, it is not necessary to quote characters `#` and `[` unless they are together. For example: `to create an expression just write \#[ the-expression-to-resolve ]`. 
+Parse Template supports the use of nested expression and literals inside expressions. There are some characters for special uses:
+
+* An expression, which begins with the sequence `+#[+`, contains nested expressions. Inside an expression, the character `[` defines the start of an expression. The character  `[` does not need a preceding `+#+` . The character `[` is treated as a special character inside an expression.
+* Inside an expression, single quotes are used both to quote text and to treat text as a literal. So, the expression `+#['[']+` is a valid expression that evaluates to `[`. Outside expressions, single quotes are used as literals.
+* To avoid using a single quote to quote text inside an expression, add a backslash to escape it. For example `#['this isn\'t an expression']`.
```

Comment:
the backslash is not rendered in the preview, please check locally if it is displayed.